{{cookiecutter.project_slug}}/k8s/prod/kustomization.yaml

```diff
@@ -7,3 +7,6 @@ bases:
 
 patches:
  - path: patches.yaml
+
+patchesStrategicMerge:
+ - patch-delete-mailhog.yaml
```

Comment:
This is fine if we have to, but "patchesStrategicMerge" is deprecated and this should work with minor changes in the "patches" section. Try running `kustomize edit fix` to see what it suggests.